### PR TITLE
Add review button link in event page

### PR DESCRIPTION
**Pull request by puranban**
## Changes

* changes in the event page
  - Add review link button
  - Show review progress status
* Remove unused styling class name

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [ ] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments

### CODE REVIEW

File: src/views/Event/index.tsx [hunk 1]
```diff
@@ -134,13 +142,27 @@ function Event(props: EventProps) {
         <div className={_cs(styles.event, className)}>
             <PageHeader
                 title={title}
+                icons={(
+                    <Status status={eventData?.event?.reviewStatus} />
+                )}
```
Comment (tnagorra) [inline src/views/Event/index.tsx:147]: Let's only pass icons when the reviewStatus is defined

File: src/views/Event/index.tsx [hunk 2]
```diff
@@ -134,13 +142,27 @@ function Event(props: EventProps) {
...
                 contentClassName={styles.details}
                 heading="Details"
                 headerActions={eventData?.event?.id && (
                     <>
+                        {(eventData.event.assignee?.id === user?.id && (
+                            <ButtonLikeLink
+                                title="Review"
+                                route={route.eventReview}
+                                attrs={{ eventId }}
+                                disabled={loading}
```
Comment (tnagorra) [inline src/views/Event/index.tsx:160]: We do not need to disable the link

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/views/Event/index.tsx (hunk 1 above)
Comment (puranban) [reply src/views/Event/index.tsx]: done

File: src/views/Event/index.tsx (hunk 2 above)
Comment (puranban) [reply src/views/Event/index.tsx]: done

**Review by tnagorra — APPROVED** [review]

**Pull request merged**